apps/namadillo/src/hooks/useWalletManager.tsx

```diff
@@ -37,6 +42,35 @@ export const useWalletManager = (wallet: WalletConnector): UseWalletOutput => {
     })();
   }, [isConnected, walletKey, chainId]);
 
+  const connectAllKeplrChains = async (): Promise<void> => {
```

Comment:
IIRC this hook was created to connect to different wallets(not only keplr). Maybe it makes sense to move this fn to keplr manager?